.github/workflows/format.yml

```diff
@@ -0,0 +1,30 @@
+name: Format suggestions
+
+on:
+  pull_request:
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: julia-actions/setup-julia@19bd4db8c427e07d56602f1c432e1a26a55d6735 # v1.8.2
+        with:
+          version: 1
+      - shell: julia --color=yes {0}
+      - run: |
+          using Pkg
+          Pkg.add(PackageSpec(name="JuliaFormatter", version="1.0.10"))
+          using JuliaFormatter
+          format(".", MinimalStyle(); normalize_line_endings = "unix")
```

Comment:
Would it make sense to explicitly state the dirs to `format(["src", "stdlib", "test"], ...)`.